### PR TITLE
fix deserialization bugs for unsupported enumerate values

### PR DESCRIPTION
**Pull request by jinyangyang222**
related to #667

### CODE REVIEW

**Comment by jinyangyang222** [comment]
@swaroopar Hi swaroopar, Please review this, if this small modification is ok, I will fix all enums and tests.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/common/exceptions/UnsupportedEnumValueException.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.common.exceptions;
+
+/**
+ * Exception thrown when there is an error during enum serialization.
```
Comment (swaroopar) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/common/exceptions/UnsupportedEnumValueException.java:9]: Should be deseralization.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/common/exceptions/UnsupportedEnumValueException.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ */
+public class UnsupportedEnumValueException extends RuntimeException {
+
+    public UnsupportedEnumValueException(String message) {
+        super("Enums serialization error:" + message);
```
Comment (swaroopar) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/common/exceptions/UnsupportedEnumValueException.java:14]: No need of this extra text. Just pass the ''message" string.

**Review by swaroopar — CHANGES_REQUESTED** [review]
@jinyangyang222 looks good. Just minor comment added. We must still add exception handler for this new exception. Either with this PR or the next PRs for the same topic.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/common/exceptions/UnsupportedEnumValueException.java (hunk 2 above)
Comment (jinyangyang222) [reply modules/models/src/main/java/org/eclipse/xpanse/modules/models/common/exceptions/UnsupportedEnumValueException.java:14]: ok.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/common/exceptions/UnsupportedEnumValueException.java (hunk 1 above)
Comment (jinyangyang222) [reply modules/models/src/main/java/org/eclipse/xpanse/modules/models/common/exceptions/UnsupportedEnumValueException.java:9]: okay.

**Comment by jinyangyang222** [comment]
> @jinyangyang222 looks good. Just minor comment added. We must still add exception handler for this new exception. Either with this PR or the next PRs for the same topic.

okay,exception handler added.

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**